conf/testOnlyDoNotUseInAppConf.routes

```diff
@@ -10,4 +10,8 @@
 # Failing to follow this rule may result in test routes deployed in production.
 
 # Add all the application routes to the prod.routes file
+#
+GET     /help-to-save/test-only/populate-reminders/:noUsers/:emailPrefix/daysToReceive        uk.gov.hmrc.helptosavefrontend.controllers.test.TestController.populateHTSReminders(noUsers: Int, emailPrefix: String, day: List[Int])
+GET     /help-to-save-reminder/test-only/gethtsuser/:nino                                     uk.gov.hmrc.helptosavefrontend.controllers.test.TestController.getHTSUser(nino: String)
```

Comment:
I don't think this will be found in QA. You need to keep the base url as the frontend one.

Reply:
In theory both should start with /help-to-save/ if that's the frontend first part of the url.